mailbox/lucene/src/test/java/org/apache/james/mailbox/lucene/search/LuceneFSSearchHighLighterTest.java

```diff
@@ -65,7 +66,7 @@ public void setUp(@TempDir Path tempPath) throws Exception {
             .listeningSearchIndex(Throwing.function(preInstanciationStage -> new LuceneMessageSearchIndex(
                 preInstanciationStage.getMapperFactory(), new InMemoryId.Factory(), FSDirectory.open(tempPath),
                 messageIdFactory,
-                preInstanciationStage.getSessionProvider())))
+                preInstanciationStage.getSessionProvider(), new JsoupTextExtractor())))
```

Comment:
Cool but it will likely create Spring issues, correct?

We need to fixup spring injects as well (inject JsoupTextExtractor is enough?)